src/styles/_modal.scss

```diff
@@ -0,0 +1,48 @@
+/*
+  <article class="modal-container modal-container--with-space">
```

Comment:
Code may be deleted?

Reply:
This comment show the expected markup for this CSS rules.
Why not keeping it ?

Reply:
What if markup changes and comment isn't updated? Markup is already in tsx file 🤔

Reply:
done